docs/static/includes/interfaces/int.pe.schema.tf

```diff
@@ -1,5 +1,17 @@
 # In this example we only support one service, e.g. Key Vault.
 # If your service has multiple private endpoint services, then expose the service name.
+
+# This variable is used to determine if the private_dns_zone_group block should be included,
+# or if it is to be managed externally, e.g. using Azure Policy.
+# https://github.com/Azure/terraform-azurerm-avm-res-keyvault-vault/issues/32
+# Alternatively you can use AzAPI, which does not have this issue.
+variable "private_endpoints_manage_dns_zone_group" {
+  type        = bool
+  default     = true
+  nullable    = false
+  description = "Whether to manage private DNS zone groups with this module. If set to false, you must manage private DNS zone groups externally, e.g. using Azure Policy."
+}
```

Comment:
With the variable kept outside of the map would this prevent the end user from being able to choose on a private endpoint by private endpoint basis whether the endpoint manages the DNS zone or not?

Reply:
Yes. However the use case is azure policy, which would either apply to the resource or it wouldn't.

Is there another use case that we are missing?